src/jupyter-aou/docker-compose.yaml

```diff
@@ -0,0 +1,50 @@
+version: '2.4'
+services:
+  app:
+    container_name: "application-server"
+    image: "gcr.io/deeplearning-platform-release/tf-gpu.2-11.py310"
+    restart: always
+    volumes:
+      - .:/workspace:cached
+      - work:/home/jovyan/work
+      - gcsfuse-mount:/mnt/gcs
+    ports:
+      - '8888:8888'
+    environment:
+      JUPYTER_ENABLE_LAB: "yes"
+    command: "start-notebook.py --ServerApp.token='' --ServerApp.password='' --ServerApp.root_dir=/home/jovyan/work --ServerApp.allow_origin='*'"
+    networks:
+      - app-network
+    devices:
+      - /dev/fuse
+    security_opt:
+      - apparmor:unconfined
+  wondershaper:
+    container_name: "wondershaper"
+    image: "us.gcr.io/broad-dsp-gcr-public/wondershaper:0.0.1"
```

Comment:
Wondershaper for rate limiting. Eventually, it's better to host this image in a VWB's public repo